src/python/grpcio/grpc/experimental/gevent.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2018 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""gRPC's Python gEvent APIs"""
+
+from grpc._cython import cygrpc as _cygrpc
+
+
+def init_gevent():
+    """Patches gRPC's libraries to be compatible with gevent.
+
+  This must be called AFTER the python standard lib has been patched,
```

Comment:
This line (and the other two-space-indented lines in this doc string) should be four-spaces-indented.

I think this is one of those things where `yapf` isn't yet smart enough to do it, but mercifully it is smart enough to avoid undoing it.

Reply:
Changed.